src/os_posix/os_union_fs.c

```diff
@@ -455,41 +476,24 @@ __union_fs_free_extent_list(WT_SESSION_IMPL *session, WT_UNION_FILE_HANDLE *unio
 static int
 __union_fs_fill_holes_on_file_close(WT_FILE_HANDLE *file_handle, WT_SESSION *wt_session)
 {
-    WT_UNION_ALLOC_LIST *alloc, *next_alloc;
     WT_UNION_FILE_HANDLE *fh;
-    char buf[4096000]; // FIXME - 4MB buffer as a placeholder
-    wt_off_t file_size;
+    WT_UNION_HOLE_LIST *hole;
+    /*
+     * FIXME - 4MB buffer as a placeholder. When we find a large hole we should break the read into
```

Comment:
Can you add the ticket: WT-13810 to this fixme?

Reply:
[9f8e50e](https://github.com/wiredtiger/wiredtiger/pull/11268/commits/9f8e50e9e9e25c96dd296661234032fd94c5c45c)